Header header
# Angle and cost of best-fitting contour
float64 angle
float64 cost